### PR TITLE
Fix python3 errors in pfcwd test suite

### PR DESCRIPTION
**Pull request by SavchukRomanLv**
### Description of PR

Summary:
When run pfcwd test suite using python3 following error appears: 
`TypeError: exec_command() got an unexpected keyword argument 'template' 
           fatal: [XX]: FAILED! => { 
               "msg": "Unexpected failure during module execution.", 
               "stdout": "" 
           } `

And when fix provided exec_template.yml:
`             Traceback (most recent call last):  File "/usr/local/lib/python2.7/dist-packages/ansible/executor/task_executor.py", line 145, in run
                 res = self._execute()
                File "/usr/local/lib/python2.7/dist-packages/ansible/executor/task_executor.py", line 664, in _execute
                  result = self._handler.run(task_vars=variables)
              File "/root/mars/workspace/sonic-mgmt/ansible/plugins/action/apswitch.py", line 71, in run
                   os_name=_os_name)
                 File "/root/mars/workspace/sonic-mgmt/ansible/plugins/connection/switch.py", line 253, in exec_command
                   before = self._remove_unprintable(client.before.decode())
              UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 47: ordinal not in range(128)` 

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
Run pfcwd TC using python3
#### How did you do it?
Add fix in swith.py and exec_tamplate.yml
#### How did you verify/test it?
Run pfcwd TC: `/env-python3/bin/python3 -m pytest pfcwd/ --inventory="../ansible/inventory,../ansible/veos" --host-pattern  --module-path                ../ansible/library/ --testbed  --testbed_file ../ansible/testbed.csv                --allow_recover --assert plain --log-cli-level debug --show-capture=no -ra --showlocals -v \ --topology t0,any,util `

All TC passed

### CODE REVIEW

**Review by roy-sror — APPROVED** [review]

**Comment by SavchukRomanLv** [comment]
Hi @wangxin @yxieca can you take a look?

File: tests/scripts/exec_template.yml [hunk 1]
```diff
@@ -12,3 +12,5 @@
       args:
         login: "{{ switch_login[hwsku_map[peer_hwsku]] }}"
       connection: switch
+      vars:
+        ansible_connection: switch
```
Comment (yxieca) [inline tests/scripts/exec_template.yml:16]: @SavchukRomanLv , Can you add PR comment to describe why this change is needed?

Comment (SavchukRomanLv) [reply tests/scripts/exec_template.yml:16]: @yxieca we had similar issue with such errors:

TypeError: exec_command() got an unexpected keyword argument 'template'  fatal: [XX]: FAILED! => {  "msg": "Unexpected failure during module execution.",  "stdout": ""  }

 https://github.com/sonic-net/sonic-mgmt/issues/2387

Comment (yxieca) [reply tests/scripts/exec_template.yml:16]: @SavchukRomanLv thanks for address this issue? Is this issue also caused by 'python3' which this PR's title is referencing fixing?

Both changes look good to me, I just don't think they should be in a single PR. Would you mind splitting them into 2 PRs?

**Comment by liat-grozovik** [comment]
@yxieca please help to close the loop of the review

**Comment by yxieca** [comment]
Merged split PRs.
